src/BitcoinHelpers.js

```diff
@@ -576,8 +618,6 @@ const BitcoinHelpers = {
      * by the `bitcoinAddress`.
      *
      * @param {string} bitcoinAddress Bitcoin address to check.
-     *
-     * @return {Promise<Array>} A promise to the found array of transactions.
```

Comment:
Should this be `@return {Promise<TransactionInBlock[]}` rather than removed?

Reply:
I removed it here because TypeScript infers it from the simplicity of the internal invocation. I can add it back if that feels better though.

Reply:
Fixed upon further reflection.